### PR TITLE
enabled no important rule

### LINKED ISSUE #573: Maintainable CSS != !important rule
We use a lot of `!importants` so far. IMHO that is a big issue. Important is only needed if you can anwers this question:
Do we EVER want to override the attribut? if answer is yes, DONT use important.

### PR DESCRIPTION
**Pull request by AndyOGo**
Fixes #573 .

Changes proposed in this pull request:

 - adds [`declaration-no-important`](https://stylelint.io/user-guide/rules/declaration-no-important/)
 - fixes `!important`s

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
 
 # How Has This Been Tested?
 
 
 
 # Checklist:
 
 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have made corresponding changes to the documentation
 - [x] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AndyOGo** [comment]
@LucaMele 
So I'm basically done. I tested it locally.

Edge would definitely need to be tested as mentioned above.

**Comment by LucaMele** [comment]
EXPECTED: 
bildschirmfoto 2018-10-04 um 09 19 35

**Comment by LucaMele** [comment]
apart of that, on ie11, looks good :) Thank you @AndyOGo

File: src/components/m-footer-links/index.scss [hunk 1]
```diff
@@ -98,6 +98,7 @@
   }
 
   @include respond-up(sm) {
+    /* stylelint-disable-next-line declaration-no-important */
     height: auto !important;
```
Comment (LucaMele) [inline src/components/m-footer-links/index.scss]: i thinks this is dangerous, but ok lets leave it

**Comment by AndyOGo** [comment]
thanks a lot for your review, will fix those issues

**Comment by AndyOGo** [comment]
@LucaMele 
both should be fixed now:)

**Comment by AndyOGo** [comment]
@LucaMele 
Alright, I added him to the list of reviewers - hope he can do it asap.
Next I would really like to have my proptypes PR reviewed #585 💪

**Comment by LucaMele** [comment]
before daily he was on it so it should be this morning :)

**Pull request merged**